src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Device.Pwm
```

Comment:
Given the folder structure, didn't we agree that this would be under Channels namespace as well? Not super important as it is not public facing but still might be worth it in order to follow convention.

Reply:
It is updated in last commit